### DISCUSSION TITLE
How to disable keepalive for a cluster

### DISCUSSION THREAD

**Discussion by doddgu** [dotnet/yarp#2424] [Q&A] [answered]
Sometimes, I want to disable keepalive for a cluster.

**Comment by doddgu** [comment]
I use transform to set response headers `Connection: close`, but I don't know if it 's a best practice.

**Reply by MihaZupan** [reply, marked as answer]
Do you mean that you want clients connecting to the proxy to close connections sooner, or that you don't want to keep persistent connections between the proxy and the backend?

If it's the former, including a connection close header on responses like you've mentioned seems reasonable.

**Reply by doddgu** [reply]
Yes, I want clients connectin to the proxy to close connections sooner. Thanks for your answer.
